### ISSUE TITLE
The "options.context" property must be an vm.Context.

### ISSUE THREAD

**Issue by unional** [kayahr/jest-electron-runner#33]
```sh
TypeError: The "options.context" property must be an vm.Context. Received an instance of Object

          at Runtime.loadEsmModule (../../node_modules/.pnpm/jest-runtime@29.7.0/node_modules/jest-runtime/build/index.js:516:20) 
```

Getting this error when using `NODE_OPTIONS=--experimental-vm-modules`.

It could be a non-issue as electron doesn't really support ESM yet (although latest version starts supporting it),
but opening this issue for reference.

**Comment by kayahr** [comment]
I don't think that it will magically work out of the box by just upgrading Electron in the future. jest-electron-runner is pretty much tied to CJS require calls and I don't think I will be able to fix this. But pull requests are always welcome.

Actually for ESM projects I strongly recommend abandoning Jest and jest-electron-runner and switch to [Vitest](https://vitest.dev/) and use its [Browser Mode](https://vitest.dev/guide/browser/) to run tests in a headless Chromium.

**Comment by kayahr** [comment]
I'm closing this issue because there will be no fix. I'm no longer using Jest, switched to Vitest and I'm going to archive this project. If someone wants to take over, feel free to fork it.
